src/data-table/data-table.js

```diff
@@ -84,6 +83,128 @@ function useSortParameters() {
   return [sortIndex, sortDirection, handleSort];
 }
 
+const HeaderContext = React.createContext<{
+  addFilter: (*, string, string) => void,
```

Comment:
Afaik `*` are now just aliased to `any`, so we should use `any`s?

Reply:
updated